kotlin/internal/jvm/impl.bzl

```diff
@@ -201,3 +201,6 @@ def kt_jvm_junit_test_impl(ctx):
             direct = ctx.files._java_runtime,
         ),
     )
+
+def kt_plugin_impl(ctx):
```

Comment:
Export the appropriate provider to be consumed by the kt_* rules

Reply:
By exporting, do you mean returning as part of the list? If so, I did this